packages/consumption/test/modules/requests/RequestsIntegrationTest.ts

```diff
@@ -104,6 +113,7 @@ export class RequestsTestsContext {
             return originalCanCreate.call(context.outgoingRequestsController, params);
         };
 
+        context.relationshipToReturnFromGetRelationshipToIdentity = TestObjectFactory.createRelationship();
```

Comment:
I'd highly vote for undoing that and putting it everywhere required. Just assuming for every test that there is a relationship breaks the whole concept.

@tnotheis what's your opinion on that?

Reply:
Yeah, I agree with @jkoenig134.

Reply:
alright